.github/workflows/ci.yml

```diff
@@ -629,6 +629,7 @@ jobs:
             -Dtesting.azure-abfs-container="${ABFS_CONTAINER}" \
             -Dtesting.azure-abfs-account="${ABFS_ACCOUNT}" \
             -Dtesting.azure-abfs-access-key="${ABFS_ACCESSKEY}" \
+            -Dtesting.aws.region="${AWS_REGION}" \
```

Comment:
Can we use AWS_REGION directly in test code?

it would be easier to run the tests from IDE if we settle on just one thing to set.